packages/image/src/__test__/load.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { type ImageSource, load } from "..";
+
+class MockImage {
+  public src = "";
+
+  public setAttribute(name: string, value: string) {
+    this[name] = value;
+  }
+
+  public constructor() {
+    MockImage.lastInstance = this;
+  }
+
+  public static lastInstance: MockImage;
+}
```

Comment:
The `MockImage` was utilized in the tests to substitute the default implementation of `window.Image`. This was done because directly using the actual browser's Image object led to test failures, hence the need for mocking to address this.

any good alternatives?

Reply:
https://github.com/fepack/image/pull/32/files#r1333379413